test/app-db/src/main/java/org/glassfish/soteria/test/ApplicationConfig.java

```diff
@@ -42,12 +42,13 @@
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Named;
 import javax.security.enterprise.identitystore.DatabaseIdentityStoreDefinition;
+import javax.security.enterprise.identitystore.Pbkdf2PasswordHash;
 
 @DatabaseIdentityStoreDefinition(
     dataSourceLookup="${'java:global/MyDS'}", 
     callerQuery="#{'select password from caller where name = ?'}",
     groupsQuery="select group_name from caller_groups where caller_name = ?",
-    hashAlgorithm = TestPlaintextPasswordHash.class,
+    hashAlgorithm = Pbkdf2PasswordHash.class,
     hashAlgorithmParameters = {
```

Comment:
In this case, the parameters are standard and always the same for all implementations of Pbkdf2PasswordHash, per the spec.